Classes/Repository/RepositoryEmptyResultsCell.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  RepositoryEmptyResultsCell.swift
+//  Freetime
+//
+//  Created by Sherlock, James on 01/08/2017.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+import IGListKit
+
+enum EmptyResultsType {
+    case issues
+    case pullRequests
+    
+    var icon: UIImage? {
+        switch self {
+        case .issues: return UIImage(named: "issue-opened")
+        case .pullRequests: return UIImage(named: "git-pull-request")
+        }
+    }
+    
+    var text: String {
+        switch self {
+        case .issues: return NSLocalizedString("This repository currently has no issues!", comment: "")
+        case .pullRequests: return NSLocalizedString("This repository currently has no pull requests!", comment: "")
+        }
+    }
+    
+    func diffIdentifier() -> NSObjectProtocol {
+        switch self {
+        case .issues: return "issues" as NSObjectProtocol
+        case .pullRequests: return "pullRequests" as NSObjectProtocol
+        }
+    }
+    
+    func isEqual(toDiffableObject object: ListDiffable?) -> Bool {
+        guard let object = object as? EmptyResultsType else { return false }
+        return self == object
+    }
+}
+
+final class RepositoryEmptyResultsCell: UICollectionViewCell {
+    
+    var type: EmptyResultsType? {
+        didSet {
+            icon.image = type?.icon?.withRenderingMode(.alwaysTemplate)
+            label.text = type?.text
+        }
+    }
+    
+    private let icon = UIImageView()
+    private let label = UILabel()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        accessibilityTraits = UIAccessibilityTraitStaticText
```

Comment:
Awesome! Will do a more thorough review of accessibility when I find the time.

Also, please use `|=` as to not cancel out any previous traits.